SRC/PORT.CPP

```diff
@@ -134,6 +155,19 @@ static void tk_port_init_time()
 #endif
 }
 
+void tk_port_resolution_changed( void )
+{
+    SDL_RenderClear( renderer );
+    get_render_rect( &render_dest_rect );
+
+    SDL_FreeSurface( surface );
```

Comment:
Would feel cleaner to have `create_surface()` free the old surface if there is one. That way there's no way the old surface can leak.

Minor nit though, no biggie.